etl/src/clients/bigquery.rs

```diff
@@ -579,415 +574,6 @@ impl BigQueryClient {
     }
 }
 
-impl Message for TableRow {
```

Comment:
All of this code has been lifted into places to be shared between the new and old client but everything will be unified once we fade out all v1 code.